### PR TITLE
SERVER-224: Added create classroom page

### PR DESCRIPTION
**Pull request by shangerdi**

### CODE REVIEW

File: server/staff/urls.py [hunk 1]
```diff
@@ -39,5 +39,6 @@
     url(r'^school/account/income/records/$', views.SchoolIncomeRecordView.as_view(), name="school_income_records"),
     url(r'^school/account/income/audit/$', views.SchoolIncomeAuditView.as_view(), name="school_income_audit"),
     url(r'^region/(?P<rid>\d+)/$', views.RegionView.as_view(), name="region_config"),
-    url(r'^course/live-course/$', views.LiveCourseView.as_view(), name="live_course")
+    url(r'^course/live-course/$', views.LiveCourseView.as_view(), name="live_course"),
+    url(r'^course/live-course/classroom/create$', views.CreateClassRoomView.as_view(), name="create_room"),
```
Comment (1e0ng) [inline server/staff/urls.py]: It's better not to use verbal like `create` in url, because a url is a resource.

File: server/staff/views.py [hunk 2]
```diff
@@ -2842,3 +2842,31 @@ def get_context_data(self, **kwargs):
...
+        capacity = parseInt(request.POST.get('capacity'), 0)
+
+        if not name:
+            return JsonResponse({'ok': False, 'msg': '请输入教室名称'})
+        if not capacity > 0:
+            return JsonResponse({'ok': False, 'msg': '请输入有效学生数量'})
+
+        if models.ClassRoom.objects.filter(school=school, name=name).exists():
+            return JsonResponse({'ok': False, 'msg': '创建失败, 教室名重复'})
+
```
Comment (1e0ng) [inline server/staff/views.py:2867]: Currently there should be at most one classroom for each school, so here we can just check if there exists a classroom in this school.  `if models.ClassRoom.objects.filter(school=school).exists()`

File: server/staff/urls.py (hunk 1 above)
Comment (shangerdi) [reply server/staff/urls.py]: removed create

File: server/staff/views.py (hunk 2 above)
Comment (shangerdi) [reply server/staff/views.py:2867]: fixed

**Comment by leonsim** [comment]
LGTM

**Comment by 1e0ng** [comment]
LGTM

**Pull request merged**